spec/swagger/v1/schemas/ingredient_schema.rb

```diff
@@ -0,0 +1,72 @@
+INGREDIENT_SCHEMA = {
+  type: :object,
+  properties: {
+    provider_id: { type: :integer, example: 666, 'x-nullable': true },
+    name: { type: :string, example: 'Some name', 'x-nullable': true },
+    sku: { type: :string, example: 'Some sku', 'x-nullable': true },
+    price: { type: :integer, example: 666, 'x-nullable': true },
+    currency: { type: :string, example: 'Some currency', 'x-nullable': true },
+    quantity: { type: :integer, example: 666, 'x-nullable': true },
+    measure: { type: :string, example: 'Some measure', 'x-nullable': true },
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Style/TrailingCommaInHashLiteral: Avoid comma after the last item of a hash.